src/components/page-intro.jsx

```diff
@@ -41,7 +36,9 @@ results
           )}
         </h2>
 
-        <div className="intro-content" style={{ display: showContent ? 'block' : 'none' }}>
+        <div
+          className={showContent ? 'intro-content intro-content--display-content' : 'intro-content'}
```

Comment:
Just a suggestion, up to you if you want to make it this way or not:
    className={`intro-content ${showContent && 'intro-content--display-content'}`}